meinberlin/apps/budgeting/templatetags/react_proposals.py

```diff
@@ -4,14 +4,21 @@
 from django.urls import reverse
 from django.utils.html import format_html
 
+from adhocracy4.phases.predicates import has_feature_active
+from meinberlin.apps.budgeting.models import Proposal
+
 register = template.Library()
 
 
 @register.simple_tag(takes_context=True)
-def react_proposals(context, obj):
-    proposals_api_url = reverse('proposals-list', kwargs={'module_pk': obj.pk})
+def react_proposals(context, module):
+    proposals_api_url = reverse('proposals-list',
+                                kwargs={'module_pk': module.pk})
     attributes = {'proposals_api_url': proposals_api_url,
-                  'is_voting_phase': 'voting' in obj.active_phase.type}
+                  'is_voting_phase': has_feature_active(module,
```

Comment:
@fuzzylogic2000 I fixed this for now, so we dont get the 500, but I think this should go to the api as has_voting_permission, so admins are always allowed to vote, right? What do we do with showing the ratings though? Should they be visible for admins all the time? So then also just show them with has_rating_permission?

Reply:
nice